packages/wrangler/src/deploy/index.ts

```diff
@@ -211,6 +211,17 @@ export const deployCommand = createCommand({
 			hidden: true,
 			alias: "x-auto-create",
 		},
+		tag: {
+			describe: "A tag for this Worker Gradual Rollouts Version",
+			type: "string",
+			requiresArg: true,
+		},
+		message: {
+			describe:
+				"A descriptive message for this Worker Gradual Rollouts Version",
+			type: "string",
+			requiresArg: true,
+		},
```

Comment:
The product name is gradual deployments, I think (https://developers.cloudflare.com/workers/configuration/versions-and-deployments/gradual-deployments/). However, given this is for `deploy`, why the mention of versions?